telescope/query.py

```diff
@@ -135,7 +135,7 @@ def _create_select_clauses(metric):
 class BigQueryQueryGenerator(object):
 
     database_name = 'plx.google'
-    table_format = '[{database_name}:m_lab.{table_date}.all]'
```

Comment:
(kind of out of scope, but as long as we're touching it) The `database_name` is always `plx.google` which means that the table name is constant and does not need to be constructed out of a format string. I suggest changing the call to `built_query_format.format` to just be:

`table='plx.google:m_lab.ndt.all'`

and deleting other table variables (`table_format`, `self._table`).